plugins/aws_sts/lib/samson_aws_sts/samson_plugin.rb

```diff
@@ -72,8 +72,8 @@ def session_name(deploy)
   ]
 end
 
-Samson::Hooks.callback :deploy_env do |deploy|
-  next {} if deploy.stage.aws_sts_iam_role_arn.blank?
+Samson::Hooks.callback :deploy_env do |deploy, *|
```

Comment:
_deploy_group would be clearer what the second variable you're expecting is...

Reply:
added